src/config.rs

```diff
@@ -152,6 +170,22 @@ impl Config {
                                 add_link_args( &mut config, backend, link_args.clone() )?;
                             }
                         },
+                        "mount-path" => {
+                            let mount_path: String =
+                                toplevel_value.try_into().map_err( |_|
+                                    format!( "{}: 'mount-path' is not a string", config.source()
+                                ))?;
+
+                            if mount_path.chars().last().unwrap() != '/' {
+                                warnings.push( Warning::Custom(
+                                    "mount-path should end with a slash".to_owned()
+                                ));
+                            }
+
+                            for backend in ALL_BACKENDS.iter().cloned() {
+                                add_mount_path( &mut config, backend, mount_path.clone() )?;
+                            }
+                        },
```

Comment:
I'd like this to work for all of the targets (not only for `wasm32-unknown-unknown`), but I'm fine with only having it work for `wasm32-unknown-unknown` for now, provided this limitation will actually be explicit. That is - you can leave all of this code here, but add an extra checks which will make sure that:

1. A non per-target `mount-path` will be rejected,
2. Only a `wasm32-unknown-unknown`-specific one will be accepted.

Again, we can leave the most of this code as-is (since I would like to support this on all targets in the future), just make sure there will be no confusion to the user that the attribute is currently supported when it isn't.